### PR TITLE
solved the repeatStringNumTimes

### PR DESCRIPTION
**Pull request by iamfortune**
used an if statement with the native javascript repeat function, hope that is okay?

### CODE REVIEW

File: daily-coding-challenges/24-06-2020/repeatString.js [hunk 1]
```diff
@@ -13,7 +13,10 @@
 
 const repeatStringNumTimes = (str, num) => {
 	/* Your implementation goes here */
-	return str;
+	if(num > 0) 
+	return str.repeat(num);
```
Comment (meekg33k) [inline daily-coding-challenges/24-06-2020/repeatString.js:20]: I like this.. this shows good knowledge of JavaScript APIs.

File: daily-coding-challenges/24-06-2020/repeatString.js [hunk 2]
```diff
@@ -13,7 +13,10 @@
 
 const repeatStringNumTimes = (str, num) => {
 	/* Your implementation goes here */
-	return str;
+	if(num > 0)
```
Comment (meekg33k) [inline daily-coding-challenges/24-06-2020/repeatString.js:19]: @iamfortune, What happens if `num` is equal to 0. 
For example if I do `repeatStringNumTimes('abc', 0)`, what should it return? There is one more case you're missing here

**Review by meekg33k — CHANGES_REQUESTED** [review]

File: daily-coding-challenges/24-06-2020/repeatString.js (hunk 1 above)
Comment (iamfortune) [reply daily-coding-challenges/24-06-2020/repeatString.js:20]: had to read up, thanks to you

File: daily-coding-challenges/24-06-2020/repeatString.js (hunk 2 above)
Comment (iamfortune) [reply daily-coding-challenges/24-06-2020/repeatString.js:19]: and edge case... hmm

Comment (iamfortune) [reply daily-coding-challenges/24-06-2020/repeatString.js:19]: solved it

File: daily-coding-challenges/24-06-2020/repeatString.js [hunk 3]
```diff
@@ -13,6 +13,9 @@
 
 const repeatStringNumTimes = (str, num) => {
 	/* Your implementation goes here */
+	if(num === 0) {
+		return null
```
Comment (meekg33k) [inline daily-coding-challenges/24-06-2020/repeatString.js]: @iamfortune, this is good but if `num` is zero, ideally you want to return the string not `null` right?

**Pull request merged**